torch/_C/__init__.pyi.in

```diff
@@ -1821,6 +1821,7 @@ def _cuda_cudaHostAllocator() -> _int: ...
 def _cuda_cudaCachingAllocator_raw_alloc(size: _int, cuda_stream: _int) -> _int: ...
 def _cuda_cudaCachingAllocator_raw_delete(ptr: _int) -> None: ...
 def _cuda_cudaCachingAllocator_set_allocator_settings(env: str) -> None: ...
+def _cuda_beginAllocateToPool(device: _int, mempool_id: Tuple[_int, _int]) -> None: ...
```

Comment:
This is confusing, you still end the allocations still with `_cuda_endAllocateCurrentStreamToPool`? What exactly is the difference?